docs/sphinx/install.rst

```diff
@@ -132,6 +132,11 @@ for all releases `here <https://github.com/NVIDIA/cuda-quantum/releases>`__.
 For more information about building a CUDA Quantum Python wheel from source, see the 
 `README <https://github.com/NVIDIA/cuda-quantum/blob/main/python/README.md>`__.
 
+The CUDA Quantum Python wheels, by default, do not come installed with our GPU-enabled 
```

Comment:
The Python wheels do come with all backends now, but they require an existing CUDA installation.